corehq/apps/locations/templates/locations/location_types.html

```diff
@@ -96,6 +96,16 @@
                                           {% endblocktrans %}">
                                     </span>
                                 </th>
+                                <th data-bind="visible: advanced_mode">
+                                    {% trans "Include only" %}
+                                    <span class="hq-help-template"
+                                          data-title="{% trans "Include only" %}"
+                                          data-content="{% blocktrans %}
+                                              <p>Include only locations of the specified types.</p>
+                                              <p>Every type you select must also have its parent included.</p>
```

Comment:
Is there some way we should enforce this? 
Not having the parents included would also mean that we don't prefetch all the parents when compiling the XML, and will then do multiple queries to get the whole hierarchy of the location tree, making this pretty slow I think.

Reply:
The server-side validation of this page automatically inserts the parent types if they're not specified.